package.json

```diff
@@ -222,4 +223,4 @@
     "uuid": "^8.3.2",
     "yargs": "^17.5.1"
   }
-}
+}
```

Comment:
Reeeeeeeeeee

Reply:
maybe we should just add this file to a .eslintignore or something

Reply:
ESLint doesn't do anything for JSON files, at least, we haven't configured it to lint JSON files and it won't do so by default. Your IDE is just configured to remove trailing new lines without leaving the last new line intact.